handbook/bizops/index.md

```diff
@@ -60,17 +60,19 @@ Every Sunday night or Monday morning, [BizOps scripts](https://github.com/source
 - [All Instances](https://sourcegraph.looker.com/looks/436)<br/>
 - [Specific Instance Overview](https://sourcegraph.looker.com/dashboards/94?Unique%20Server%20ID=&Site%20ID=&filter_config=%7B%22Unique%20Server%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%22%22%7D,%7B%7D%5D,%22id%22:4%7D%5D,%22Site%20ID%22:%5B%7B%22type%22:%22%3D%22,%22values%22:%5B%7B%22constant%22:%22%22%7D,%7B%7D%5D,%22id%22:5%7D%5D%7D) (To select a specific company, fill the Unique Server ID field)<br/>
 
-**Folders**
+**Looker dashboards**
 
 - [Go-to-market](https://sourcegraph.looker.com/browse/boards/2)<br/>
 - [Marketing](https://sourcegraph.looker.com/folders/109)<br/>
```

Comment:
Should Marketing and Sales be sub-bullets here? Or — should these folder links even exist given you're using /boards/2 as the main link for both of the Marketing and Sales index.md updates?